js/index.js

```diff
@@ -26,6 +44,14 @@ function removeProduct(event) {
   const target = event.currentTarget;
   console.log('The target in remove is:', target);
   //... your code goes here
+
+
+  let removeButton = document.querySelector(".btn btn-remove")
+  removeButton.addEventListener('click', function() {
+      this.product.removeButton
+  })
+
+
 }
 
 // ITERATION 5
```

Comment:
Good work! ⭐⭐⭐ If you find the time, try to implement the functionality to remove and create a product.